modules/accredited_representative_portal/app/policies/accredited_representative_portal/authorization_policy.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class AuthorizationPolicy < ApplicationPolicy
+    def self.policy_name
+      'AccreditedRepresentativePortal::Authorization'
+    end
+
+    def authorize_as_representative?
+      # Must be logged in is enforced by ApplicationPolicy initialize
+      # Authorize only if the current user's account has at least one registration
+      account = @user.user_account
+
+      registrations = account.registrations
+      registrations.present?
```

Comment:
Should this instead be based on having any `power_of_attorney_holders`? Other authorization logic has this as its base requirement. @chumpy